src/components/media-item/index.jsx

```diff
@@ -0,0 +1,47 @@
+import PropTypes from "prop-types";
+
+const COMPONENT_CLASS_NAME = "c-media-item";
+
+const MediaItem = ({ caption, children, className, credit, title }) =>
+	title || caption || credit ? (
+		<figure
+			className={className ? `${COMPONENT_CLASS_NAME} ${className}` : `${COMPONENT_CLASS_NAME}`}
+		>
+			{children}
+			<figcaption className={`${COMPONENT_CLASS_NAME}__fig-caption`}>
```

Comment:
Do we want to render the `<figcaption>` element with no children? We should add logic to only output if there is at least one caption item to render

Reply:
updated to add the logic to only show the figcaption if there's values for the captions